src/Resources/config/block.xml

```diff
@@ -25,16 +25,12 @@
             <argument type="service" id="sonata.block.context_manager"/>
         </service>
         <service id="sonata.page.block.context_manager" class="Sonata\PageBundle\Block\BlockContextManager">
-            <argument type="service" id="sonata.block.loader.service"/>
-            <argument type="service" id="sonata.block.manager"/>
+            <argument type="service" id="sonata.block.context_manager.default"/>
         </service>
         <service id="sonata.page.block.breadcrumb" class="%sonata.page.block.breadcrumb.class%">
             <tag name="sonata.block"/>
             <tag name="sonata.breadcrumb"/>
-            <argument>page</argument>
-            <argument>sonata.page.block.breadcrumb</argument>
-            <argument type="service" id="sonata.templating"/>
-            <argument type="service" id="knp_menu.menu_provider"/>
+            <argument type="service" id="twig"/>
```

Comment:
Are all those changes correct?

Reply:
Seo has been updated, BaseBreadcrumbMenuBlockService need twig and FactoryInterface(KNP). Plus, sonata.templating disappear from the project

Reply:
where are you decorating the "SonataBlockBundle"?

Reply:
BlockContextManager & ContainerBlockService